### PR TITLE
vdrift.vdrift version 2014-10-20

### PR DESCRIPTION
**Pull request by superusercode**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107110)

### CODE REVIEW

File: manifests/v/vdrift/vdrift/2014-10-20/vdrift.vdrift.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: vdrift.vdrift
+PackageVersion: 2014-10-20
+PackageLocale: en-US
+Publisher: vdrift
+PackageName: vdrift
```
Comment (stephengillie) [inline manifests/v/vdrift/vdrift/2014-10-20/vdrift.vdrift.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: http://vdrift.net/
PublisherSupportUrl: https://github.com/VDrift/vdrift/issues
PackageName: vdrift
PackageUrl: http://vdrift.net/
```

**Comment by stephengillie** [comment]
This application installed normally, but errored when launched:
> Faulting application name: vdrift.exe, version: 0.0.0.0, time stamp: 0x00000000
Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000
Exception code: 0xc0000005
Fault offset: 0x0000000000000000
Faulting process id: 0x0x3980
Faulting application start time: 0x0x1D98938F7658E41
Faulting application path: C:\Program Files\VDrift\vdrift.exe
Faulting module path: unknown
Report Id: 95228a18-5c13-41cc-b442-df4ffcf21b9a
Faulting package full name: 
Faulting package-relative application ID:

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**